packages/docs/pages/guide/mixpanel.mdx

```diff
@@ -87,6 +87,7 @@ const growthbook = new GrowthBook({
     mixpanel.track("Experiment Viewed", {
       "Experiment Id": experiment.key,
       "Variant Id": result.variationId,
```

Comment:
We should make this match our docs when it comes to event and property names.  Mixpanel has a specific standard they use for experimentation events:

```ts
    mixpanel.track("$experiment_started", {
      "Experiment name": experiment.key,
      "Variant name":  result.variationId,
      "$source": "growthbook",
    })
```

This may break further steps in your tutorial if you are relying on other names, so I'm fine skipping this for now and updating it later.